lib/utils/color.dart

```diff
@@ -0,0 +1,12 @@
+import 'package:flutter/material.dart';
+
+class WarsColors {
+  static Color green = Colors.green[700];
+  static Color yellow = Color(0xfff99f02);
+  static Color orange = Colors.orangeAccent[700];
+  static Color pink = Colors.pink[400];
+  static Color red = Colors.red[900];
+  static Color grey = Colors.grey[700];
+  static Color blue = Color(0xff4f5096);
+  static Color greyLight = Color(0xffcfcfcf);
```

Comment:
in the future we may consider having dark mode, so maybe for each color we may have light/main/dark?